opentelemetry-api/pyproject.toml

```
@@ -29,7 +29,7 @@ dependencies = [
     "Deprecated >= 1.2.6",
     # FIXME This should be able to be removed after 3.12 is released if there is a reliable API
     # in importlib.metadata.
-    "importlib-metadata >= 6.0, <= 8.0.0",
+    "importlib-metadata >= 6.0, < 9.0.0",
```

We tend to be conservative on importlib-metadata because they don't follow semantic versioning

```suggestion
    "importlib-metadata >= 6.0, <= 8.2.0",
```

---

Dang. So there is no way around bumping this every now and then?